### PR TITLE
ign -> gz Migrate Ignition Headers : gz-rendering

### PR DESCRIPTION
**Pull request by methylDragon**
See: https://github.com/gazebo-tooling/release-tools/issues/784

### CODE REVIEW

File: examples/gazebo_scene_viewer/CameraWindow.hh [hunk 1]
```diff
@@ -14,13 +14,13 @@
...
-#define IGNITION_RENDERING_EXAMPLES_CAMERAWINDOW_HH_
+#ifndef GZ_RENDERING_EXAMPLES_CAMERAWINDOW_HH_
+#define GZ_RENDERING_EXAMPLES_CAMERAWINDOW_HH_
 
 #include <vector>
-#include "ignition/rendering/RenderTypes.hh"
+#include "gz/rendering/RenderTypes.hh"
 
-namespace gz = ignition::rendering;
+namespace ignition = gz::rendering;
```
Comment (chapulina) [inline examples/gazebo_scene_viewer/CameraWindow.hh:23]: Can this line be removed?

File: examples/gazebo_scene_viewer/GazeboDemo.cc [hunk 2]
```diff
@@ -18,19 +18,19 @@
...
 
-using namespace ignition;
+using namespace gz;
 using namespace rendering;
 
 void Connect()
 {
-  gazebo::common::Console::SetQuiet(false);
-  gazebo::transport::init();
-  gazebo::transport::run();
```
Comment (chapulina) [inline examples/gazebo_scene_viewer/GazeboDemo.cc:33]: Oh this is actually Gazebo classic

File: examples/gazebo_scene_viewer/GazeboWorldDemo.cc [hunk 3]
```diff
@@ -15,22 +15,22 @@
...
 
-using namespace ignition;
+using namespace gz;
 using namespace rendering;
 
 void Connect()
 {
   common::Console::SetVerbosity(4);
-  gazebo::transport::init();
-  gazebo::transport::run();
```
Comment (chapulina) [inline examples/gazebo_scene_viewer/GazeboWorldDemo.cc:33]: Here too Gazebo classic

File: examples/gazebo_scene_viewer/SceneManagerPrivate.hh [hunk 4]
```diff
@@ -114,27 +114,27 @@ namespace ignition
...
+      private: sim::transport::SubscriberPtr responseSub;
 
-      private: gazebo::transport::SubscriberPtr lightSub;
+      private: sim::transport::SubscriberPtr lightSub;
 
-      private: gazebo::transport::SubscriberPtr modelSub;
+      private: sim::transport::SubscriberPtr modelSub;
 
-      private: gazebo::transport::SubscriberPtr jointSub;
+      private: sim::transport::SubscriberPtr jointSub;
```
Comment (chapulina) [inline examples/gazebo_scene_viewer/SceneManagerPrivate.hh:129]: Gazebo classic

**Review by nkoenig — APPROVED** [review]

**Comment by nkoenig** [comment]
@iche033 , I think this is ready.

File: examples/gazebo_scene_viewer/GazeboDemo.cc (hunk 2 above)
Comment (iche033) [reply examples/gazebo_scene_viewer/GazeboDemo.cc:33]: oh I think these lines should not be removed. This demo shows integration with gazebo classic so the old gazebo-classic calls are expected

File: examples/gazebo_scene_viewer/SceneManagerPrivate.hh [hunk 5]
```diff
@@ -114,27 +114,27 @@ namespace ignition
 
       private: NewSceneManager *newSceneManager;
 
-      private: gazebo::transport::NodePtr transportNode;
+      private: sim::transport::NodePtr transportNode;
```
Comment (iche033) [inline examples/gazebo_scene_viewer/SceneManagerPrivate.hh:117]: these should stay `gazebo` as this demo uses gazebo-classic transport to get its scene info and render the scene using gz-rendering

File: examples/gazebo_scene_viewer/GazeboWorldDemo.cc (hunk 3 above)
Comment (iche033) [reply examples/gazebo_scene_viewer/GazeboWorldDemo.cc:33]: same here, we should keep these calls

File: examples/gazebo_scene_viewer/SceneManagerPrivate.hh (hunk 5 above)
Comment (nkoenig) [reply examples/gazebo_scene_viewer/SceneManagerPrivate.hh:117]: Ahh. I reverted these changes.

Comment (iche033) [reply examples/gazebo_scene_viewer/SceneManagerPrivate.hh:117]: I still see `sim::`, maybe the changes were not pushed?

Comment (iche033) [reply examples/gazebo_scene_viewer/SceneManagerPrivate.hh]: more `sim::` to `gazebo::` changes in [df38c4e](https://github.com/gazebosim/gz-rendering/pull/705/commits/df38c4e6cc8344c999f0d9f5ba401c9b4067f22b)

**Comment by iche033** [comment]
added more changes in [c65a4cf](https://github.com/gazebosim/gz-rendering/pull/705/commits/c65a4cff71f0935fc609a34b6de4e01201ec157b) to fix the `gazebo_scene_viewer` demo

**Review by iche033 — APPROVED** [review]

**Pull request merged**